core/object/message_queue.cpp

```diff
@@ -34,6 +34,9 @@
 #include "core/core_string_names.h"
 #include "core/object/class_db.h"
 #include "core/object/script_language.h"
+#ifdef DEBUG_ENABLED
+#include "core/config/engine.h"
+#endif
```

Comment:
Style wise, add an empty line of separation between the regular list of include and conditionally defined ones:
```suggestion

#ifdef DEBUG_ENABLED
#include "core/config/engine.h"
#endif
```